test/Carter.Tests/Modelbinding/NewtonsoftBinding/NewtonsoftTests.cs

```diff
@@ -0,0 +1,59 @@
+namespace Carter.Tests.ModelBinding.NewtonsoftBinding
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using Carter.ModelBinding;
+    using Carter.Tests.Modelbinding;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.TestHost;
+    using Microsoft.Extensions.DependencyInjection;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using Xunit;
+
+    public class NewtonsoftTests
+    {
+        private readonly HttpClient httpClient;
+
+        public NewtonsoftTests()
+        {
+            JsonConvert.DefaultSettings = () => new JsonSerializerSettings {
+                ContractResolver = new NewtonsoftJsonUtils.PrivateSetterCamelCasePropertyNamesContractResolver(),
+                Converters = new JsonConverter[] {new StringEnumConverter()},
+            };
+            
+            var server = new TestServer(
+                new WebHostBuilder()
+                    .ConfigureServices(x =>
+                    {
+                        x.AddCarter(configurator: c =>
+                            c.WithModule<NewtonsoftModule>());
```

Comment:
I wonder if we expose `WithBinder` option and provide a JSON.NET Binder too so people can use that when settings up Carter in their apps